packages/iov-bns/src/encode.ts

```diff
@@ -48,16 +48,16 @@ export const buildMsg = (tx: UnsignedTransaction): codecImpl.app.ITx => {
 
 const buildSendTx = (tx: SendTx): codecImpl.app.ITx => ({
   sendMsg: codecImpl.cash.SendMsg.create({
-    src: keyToAddress(tx.signer),
-    dest: tx.recipient,
+    src: decodeBnsAddress(keyToAddress(tx.signer)),
```

Comment:
Love all these encodeBnsAddress/decodeBnsAddress... thank you types for highlighting what to change